Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -4,19 +4,26 @@ A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-control
 
 The Leap Motion Data Provider enables articulated hand tracking for VR and could be useful for rapid prototyping in the editor.  The data provider can be configured to use the leap motion controller mounted on a headset or placed on a desk face up.
 
+![LeapMotionIntroGif](../Images/CrossPlatform/LeapMotion/LeapHandsGif3.gif)
+
 ## Using Leap Motion tracking in MRTK
 1. Prepare MRTK project for Leap Motion
 
-    - **This step only applies if the source of MRTK is cloned from the git repo and NOT the from the Unity packages. If the MRTK source is from the Unity packages, go to step 2.**
+    - **This step only applies for the Leap Motion Core Assets version 4.4.0 and if the source of MRTK is cloned from the git repo, NOT from the Unity packages. This step is not required if the Leap Motion Core Assets version 4.5.0 are used. If the MRTK source is going to be from the Unity packages, start at step 2.**
 
     - Navigate to **Mixed Reality Toolkit > Utilities > Leap Motion > Configure CSC File for Leap Motion**. Updating the csc file filters out the obsolete warnings produced by the Leap Motion Core Assets.  The MRTK repo contains a csc file that converts warnings to errors, this conversion halts the Leap Motion MRTK configuration process.  The obsolete warnings issue is tracked [here](https://github.com/leapmotion/UnityModules/issues/1082).
 
     ![LeapMotionUpdateCSCFile](../Images/CrossPlatform/LeapMotion/LeapMotionConfigureCSCFile2.png)
 
-1. Importing the Leap Motion Core Assets
-    - Import the **Microsoft.MixedReality.Toolkit.Foundation** package into the unity project.
+1. Importing MRTK and the Leap Motion Core Assets
+    - Import the **Microsoft.MixedReality.Toolkit.Foundation** package into the Unity project.
+    > [!NOTE]
+    > The Leap Motion Data Provider is supported in both Unity 2018.4.x and 2019.3.x, but 2019.3.x is preferred.
+
     - Install the [Leap Motion SDK 4.0.0](https://developer.leapmotion.com/releases/?category=orion)
-    - Download and import the [Leap Motion Core Assets 4.4.0](https://developer.leapmotion.com/unity#5436356)
+    - Download and import the [Leap Motion Core Assets for Unity](https://github.com/leapmotion/UnityModules/releases/tag/UM-4.5.0)
+        - Leap Motion Core Assets version [4.4.0](https://github.com/leapmotion/UnityModules/releases/tag/Release-CoreAsset-4.4.0) and [4.5.0](https://github.com/leapmotion/UnityModules/releases/tag/UM-4.5.0) are supported, but version 4.5.0 is preferred.
+        - If using version 4.5.0, import the **Core** package into the Unity project.
```

Comment:
This thing has a bit of text:

"- If using Unity 2019.3.x, go to step 3."

Be careful with the list we have here and explicitly called out steps - add a specific hyperlink in here if you need to reference a specific location, as explicit steps (especially in VERY long steps like this) can easily get out of date.

Reply:
I changed "go to step 3" to "go to the next step".  Since I do not think I can link a step unless it is a title that starts with '#' character